ui/templates/repository.html

```diff
@@ -66,6 +65,9 @@
 <script type="text/jsx"
         src="{% static "ui/js/utils.jsx" %}">
 </script>
+<script type="text/jsx"
+        src="{% static "ui/js/listing.jsx" %}">
```

Comment:
It matches everything else in the file, but there's no particular standard for HTML indentation in LORE